data/campaigns/Heir_To_The_Throne/utils/abilities.cfg

```diff
@@ -107,6 +107,13 @@ _"This unit's grasp of melee tactics allows adjacent allies to strike the first
                 {INITIATIVE_OBJECT x,y=$this_item.x,$this_item.y}
             [/do]
         [/foreach]
+        [lua] # Can't see why {INITIATIVE_OBJECT} leaves the last unit selected.
+            code=<<
+            wesnoth.deselect_hex()
+            -- Take THAT, you stupid bug! **SMASH**
+            -- (Gotta love the 10# sledge when you really should use a tap hammer!)
+            >>
+        [/lua]
 
```

Comment:
The question was raised yesterday on IRC: Does this effect only prestart or does it also effect units being recruited or advancing.

During tonight's re-testing after sync'ing to master, I noted that, yes, it does appear to effect units being recruited by Li'sar. My system was running slow and I could see their ellipse flashing dark (meaning they were selected by the bug then immediately de-selected by this sledgehammer. I just tested specifically for a unit advancing while standing next to Li'sar and did not notice the issue. BUT .. at that point, the unit should already have the objects. I'm not sure I can force a unit to not have the objects, but be standing next to her, and advance and get the objects but I'm thinking any time the object is added, the unit will be selected.